Example-QueryInfo-REXX/README.md

```diff
@@ -1,13 +1,13 @@
 ## Example-QueryInfo-REXX
 
 This samples uses HWIREST API to:
-- List CPCs, filtered by the CPC name provided, in order to retrieve the URI and target name associated with that CPC
+- List CPCs and retrieves the URI and target name associated with that LOCAL CPC or the CPCname provided
```

Comment:
I'm confused about singular/plural in this whole section from "This samples" to "List CPCs and retrieves"

Reply:
should be 'sample', singular

Reply:
fixed in the latest commit